website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md

```diff
@@ -3,77 +3,93 @@ title: "Migrating from dbt-spark to dbt-databricks"
 id: "migrating-from-spark-to-databricks"
 ---
 
+You can use the [dbt-databricks adapter](https://github.com/databricks/dbt-databricks) to establish a dbt connection with Databricks, instead of the `dbt-spark` adapter. In collaboration with dbt Labs, Databricks built this adapter using dbt-spark as the foundation and added some critical improvements. With it, you get an easier set up &mdash; requiring only three inputs for authentication &mdash; and more features through the Delta table format.
 
-## Pre-requisites
+## Simpler authentication
 
-In order to migrate to dbt-databricks, your project must be compatible with `dbt 1.0` or greater as dbt-databricks is not supported pre `dbt 1.0`. [This guide](https://docs.getdbt.com/guides/migration/versions/upgrading-to-v1.0) will help you upgrade your project if necessary.
+Previously, you had to provide a `cluster` or `endpoint` ID which was hard to parse from the `http_path` that you were given. Now, it doesn't matter if you're using a cluster or an SQL endpoint because the [dbt-databricks setup](/reference/warehouse-setups/databricks-setup) requires the _same_ inputs for both. All you need to provide is:
+- the hostname of the Databricks workspace
+- the HTTP path of the Databricks SQL warehouse or cluster
+- an appropriate credential
```

Comment:
Should credential be plural? If it's just the token they need (singular) then maybe we should specify they need a token?

Edit: I see it's plural later, so let's make them the same!